### DISCUSSION TITLE
Nightly Version Resolved Hanging Issue in WezTerm - How to Inform Developers?

### DISCUSSION THREAD

**Discussion by carstenuhlig** [wezterm/wezterm#5885] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows, macOS

### Which Wayland compositor or X11 Window manager(s) are you using?

none. i mean... windows caused the problems. not macos. but i also use macos.

### WezTerm version

wezterm 20240730-082727-56a27e93

### Ask your question!

I've been experiencing WezTerm crashes since Tuesday. After several hours of troubleshooting—including reinstalling, removing the config file, and trying the release version from GitHub—I finally resolved the issue by switching to the nightly version.

There were no logs generated to diagnose the problem, only leftover SOCKS files after forcefully killing the process.

Should I inform the developers that the nightly version fixed the issue? If so, where is the best place to report this? I couldn't find any guidance in the wiki or through Google searches.

Thanks for your help! (if yes, then i can provide more information of course. Just let me know)

**Comment by wez** [comment, marked as answer]
Thanks for sharing; there's nothing more to do about this if the nightly build is working fine; folks can install and run that in the meantime, until a new release is made.
